[2024-08-25T08:27:42.332+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-25T08:27:42.441+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_catchup_and_backfill_v1.task1 scheduled__2024-08-22T00:00:00+00:00 [queued]>
[2024-08-25T08:27:42.506+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_catchup_and_backfill_v1.task1 scheduled__2024-08-22T00:00:00+00:00 [queued]>
[2024-08-25T08:27:42.522+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-08-25T08:27:42.602+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): task1> on 2024-08-22 00:00:00+00:00
[2024-08-25T08:27:42.629+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1589) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-25T08:27:42.644+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_catchup_and_backfill_v1', 'task1', 'scheduled__2024-08-22T00:00:00+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_and_backfill.py', '--cfg-path', '/tmp/tmpj1pp9uvp']
[2024-08-25T08:27:42.656+0000] {standard_task_runner.py:105} INFO - Job 67: Subtask task1
[2024-08-25T08:27:42.653+0000] {standard_task_runner.py:72} INFO - Started process 1596 to run task
[2024-08-25T08:27:42.948+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_catchup_and_backfill_v1.task1 scheduled__2024-08-22T00:00:00+00:00 [running]> on host ae952724060e
[2024-08-25T08:27:43.608+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='shreyas' AIRFLOW_CTX_DAG_ID='dag_with_catchup_and_backfill_v1' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-08-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-22T00:00:00+00:00'
[2024-08-25T08:27:43.617+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-25T08:27:43.717+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-25T08:27:43.720+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo This is a simple bash command!']
[2024-08-25T08:27:43.745+0000] {subprocess.py:86} INFO - Output:
[2024-08-25T08:27:43.751+0000] {subprocess.py:93} INFO - This is a simple bash command!
[2024-08-25T08:27:43.753+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-25T08:27:43.875+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-25T08:27:43.878+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_and_backfill_v1, task_id=task1, run_id=scheduled__2024-08-22T00:00:00+00:00, execution_date=20240822T000000, start_date=20240825T082742, end_date=20240825T082743
[2024-08-25T08:27:44.055+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-25T08:27:44.109+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
